force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -122,13 +199,12 @@ public with sharing class FlowControlLoopsCollections {
      * @return The sum of all integers up to the limit.
      */
     public static Integer sumUpToLimit(Integer intLimit) {
-        /* 
-         for (){
-            
+        Integer sumOfAll = 0; 
+        Integer i;
```

Comment:
Integer I should get declared and instantiated in the for loop directly. 
`for (Integer i = 1; i <= intLimit; i++){`